sdk/eventhub/azure-eventhub/tests/pyamqp_tests/synctests/test_websocket.py

```diff
@@ -8,6 +8,7 @@
 from azure.eventhub._pyamqp import authentication, ReceiveClient
 from azure.eventhub._pyamqp.constants import TransportType
 
+@pytest.mark.skip()
```

Comment:
Skipped? Why have we skipped?

Reply:
We test the websocket transport in other tests and so we decided to skip these for now as they were breaking on the pipeline. We can remove them later and maybe add an additional test to the test_send.py in the livetests for websocket transport?